packages/web/src/shared/bibleTranslationClient.ts

```diff
@@ -0,0 +1,41 @@
+import { BibleClient } from '@gracious.tech/fetch-client';
+
+export type BibleTranslation = {
+  id: string;
+  name: string;
+};
+
+// For some reason, fetch.bible uses non-standard language codes.
+const langCodeMapping: Record<string, string> = {
+  en: 'eng',
+  es: 'spa',
+};
```

Comment:
I don't know why `fetch.bible` seems to use English-based IDs, but we're going to have to add mappings for all the common languages, otherwise our users will have to scroll through 900+ translation options, which isn't a great experience. In the future it would be great to incorporate some kind of filtering logic to allow users to search the list of available translations.

Reply:
Looks like these are using [ISO 639-3](https://en.wikipedia.org/wiki/ISO_639-3) codes, which are more precise than the two letter ISO 639-1. I think for external integrations we are probably going to need to transpose between codes, but I wonder if ISO 639-3 is a better standard for us to use to identify a language. If we are going to do that, we should do so in a separate PR